agent/util-scripts/pbench-make-result-tb

```diff
@@ -150,25 +148,25 @@ fi
 
 # if -u|--user was specified, store the specified user in metadata.log
 if [[ ! -z "$user" ]] ;then
-    printf -- "$user" | pbench-add-metalog-option ${mdlog} run user
+    printf -- "%s" "$user" | pbench-add-metalog-option ${mdlog} run user
 fi
 
 # if -p|--prefix was specified, store the specified prefix in metadata.log
 if [[ ! -z "$prefix" ]] ;then
-    printf -- "$prefix" | pbench-add-metalog-option ${mdlog} run prefix
+    printf -- "%s" "$prefix" | pbench-add-metalog-option ${mdlog} run prefix
 fi
 
 result_size=$(du -sb ${pbench_run_name} | awk '{print $1}')
 debug_log "preparing to tar up ${result_size} bytes of data from ${full_result_dir}"
-printf -- "$result_size" | pbench-add-metalog-option ${mdlog} run raw_size
+printf -- "%s" "$result_size" | pbench-add-metalog-option ${mdlog} run raw_size
 
 tarball="${target_dir}/${pbench_run_name}.tar.xz"
 if [[ "${xz_single_threaded}" == "1" ]]; then
     tar_cmd="tar --create --force-local -xz \"${pbench_run_name}\""
 else
     tar_cmd="tar --create --force-local \"${pbench_run_name}\" | xz -T0"
 fi
-printf -- "${tar_cmd} > \"${tarball}\"\n" >&2
+printf -- "%s" "${tar_cmd} > \"${tarball}\"\n" >&2
```

Comment:
It seems this should be something like:

```
printf -- "%s > \"%s\"\n" "${tar_cmd}" "${tarball}" >&2
```

This might be the reason why the unit tests a failing in TravisCI, though I am not sure.

Reply:
Ok, I will look into it